saw-remote-api/python/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
```

Comment:
For the sake of reproducibility, we should also check in the `poetry.lock` file.